eogrow/pipelines/byoc.py

```diff
@@ -109,7 +125,14 @@ def _prepare_tile(self, folder: str, tiff_paths: List[str]) -> Optional[ByocTile
         cover_geometry = self._get_tile_cover_geometry(some_tiff)
         if cover_geometry.geometry.is_empty:
             return None
-        return ByocTile(folder, cover_geometry=cover_geometry, sensing_time=self.config.sensing_time)
+
+        sensing_time = self.config.sensing_time
+        if self.config.is_temporal:
+            # assumes file-system structure is equal to that of ExportMapsPipeline
+            timestamp_folder = fs.path.split(folder)[1]
+            sensing_time = dt.datetime.strptime(timestamp_folder, TIMESTAMP_FORMAT)
```

Comment:
Since the user is the one who provides the BYOCs, shouldn't the user be able to define the timestamp format? or do you want to limit the complexity of what can be done? In that case I would make it more explicit as to what the structure is.

Reply:
just gonna answer it here since this is similar to the comment above. The pipeline currently works mostly in conjunction with the `ExportMapsPipeline`. I guess i can document the structure more, but i'm not sure where. At this point i don't feel like this should be too configurable, once we have a use-case for it we can adjust if needed.

Reply:
Perhaps it would make sense to add a short sentence in the description of the `is_temporal` parameter with an example of what is expected, just to make it more intuitive. 

In any case, not so important, feel free to ignore and resolve.

Reply:
added an example